airbyte-cdk/python/airbyte_cdk/base_python/cdk/streams/http.py

```diff
@@ -204,23 +204,23 @@ def _send_request(self, request: requests.PreparedRequest) -> requests.Response:
     def read_records(
         self,
         sync_mode: SyncMode,
-        stream_slice: Optional[Mapping[str, Any]] = None,
-        stream_state: Optional[Mapping[str, Any]] = None,
         cursor_field: List[str] = None,
+        stream_slice: Mapping[str, Any] = None,
```

Comment:
reorganised to match the `Stream` interface's `read_records` method

Reply:
👍🏼